packages/blockchain-wallet-v4-frontend/src/components/IdentityVerification/TierCard/model.js

```diff
@@ -0,0 +1,173 @@
+import React from 'react'
+import { path, prop, propEq } from 'ramda'
+import { FormattedMessage } from 'react-intl'
+import { Text } from 'blockchain-info-components'
+
+export const getTiming = (state, tier) => {
+  switch (state) {
+    case 'verified':
+      return (
+        <Text size='14px' color='success'>
+          <FormattedMessage
+            id='components.identityverification.tiercard.verified'
+            defaultMessage='Approved!'
+          />
+        </Text>
+      )
+    case 'pending':
+      return (
+        <Text size='14px' color='btc'>
+          <FormattedMessage
+            id='components.identityverification.tiercard.pending'
+            defaultMessage='In Review'
+          />
+        </Text>
+      )
+    case 'rejected':
+      return (
+        <Text size='14px' color='error'>
+          <FormattedMessage
+            id='components.identityverification.tiercard.rejected'
+            defaultMessage='Rejected'
+          />
+        </Text>
+      )
+    default:
+      return (
+        <FormattedMessage
+          id='components.identityverification.tiercard.timing'
+          defaultMessage='Takes {time} min'
+          values={{ time: path([tier, 'time'], TIERS) }}
+        />
+      )
+  }
+}
+
+export const TIERS = {
+  1: {
+    time: '3',
+    isActive: propEq('state', 'ACTIVE'),
```

Comment:
shouldn't this point to current user tier being more than tierIndex?

Reply:
Yes this was a mock I forgot to update